packages/system/data_stream/security/elasticsearch/ingest_pipeline/default.yml

```
@@ -59,10 +59,20 @@ processors:
       if: ctx?.winlog?.level != ""
   - date:
       field: winlog.time_created
+      tag: "time_created_date"
       formats:
         - ISO8601
-      ignore_failure: true
+      ignore_missing: true
       if: ctx?.winlog?.time_created != null
```

```suggestion
      if: ctx.winlog?.time_created != null
```
The presence is already handled by the condition (and `ignore_missing` is not an option for `date`). The null-safe deref is not required on the `ctx` since it is always present.

---

Ahh, that makes sense. I will correct. Thank you.